lib/WindowsCompat.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Instead of exporting an object with a method, you could just export a function directly. Maybe call it `forwardSlashes` or something to make obvious what it's doing.